gstreamer-pbutils/src/auto/functions.rs

```diff
@@ -0,0 +1,373 @@
+    }
+}
+
+pub fn missing_uri_sink_installer_detail_new(protocol: &str) -> Option<String> {
+    assert_initialized_main_thread!();
+    unsafe {
+        from_glib_full(ffi::gst_missing_uri_sink_installer_detail_new(protocol.to_glib_none().0))
+    }
+}
+
+pub fn missing_uri_sink_message_new<P: IsA<gst::Element>>(element: &P, protocol: &str) -> Option<gst::Message> {
+    assert_initialized_main_thread!();
+    unsafe {
+        from_glib_full(ffi::gst_missing_uri_sink_message_new(element.to_glib_none().0, protocol.to_glib_none().0))
+    }
+}
+
+pub fn missing_uri_source_installer_detail_new(protocol: &str) -> Option<String> {
+    assert_initialized_main_thread!();
+    unsafe {
+        from_glib_full(ffi::gst_missing_uri_source_installer_detail_new(protocol.to_glib_none().0))
+    }
+}
+
+pub fn missing_uri_source_message_new<P: IsA<gst::Element>>(element: &P, protocol: &str) -> Option<gst::Message> {
+    assert_initialized_main_thread!();
+    unsafe {
+        from_glib_full(ffi::gst_missing_uri_source_message_new(element.to_glib_none().0, protocol.to_glib_none().0))
+    }
+}
+
+pub fn pb_utils_add_codec_description_to_tag_list<'a, P: Into<Option<&'a str>>>(taglist: &gst::TagList, codec_tag: P, caps: &gst::Caps) -> bool {
+    assert_initialized_main_thread!();
+    let codec_tag = codec_tag.into();
+    let codec_tag = codec_tag.to_glib_none();
+    unsafe {
+        from_glib(ffi::gst_pb_utils_add_codec_description_to_tag_list(taglist.to_glib_none().0, codec_tag.0, caps.to_glib_none().0))
+    }
+}
+
+pub fn pb_utils_get_codec_description(caps: &gst::Caps) -> Option<String> {
+    assert_initialized_main_thread!();
+    unsafe {
+        from_glib_full(ffi::gst_pb_utils_get_codec_description(caps.to_glib_none().0))
+    }
+}
+
+pub fn pb_utils_get_decoder_description(caps: &gst::Caps) -> Option<String> {
+    assert_initialized_main_thread!();
+    unsafe {
+        from_glib_full(ffi::gst_pb_utils_get_decoder_description(caps.to_glib_none().0))
+    }
+}
+
+pub fn pb_utils_get_element_description(factory_name: &str) -> Option<String> {
+    assert_initialized_main_thread!();
+    unsafe {
+        from_glib_full(ffi::gst_pb_utils_get_element_description(factory_name.to_glib_none().0))
+    }
+}
+
+pub fn pb_utils_get_encoder_description(caps: &gst::Caps) -> Option<String> {
+    assert_initialized_main_thread!();
+    unsafe {
+        from_glib_full(ffi::gst_pb_utils_get_encoder_description(caps.to_glib_none().0))
+    }
+}
+
+pub fn pb_utils_get_sink_description(protocol: &str) -> Option<String> {
+    assert_initialized_main_thread!();
+    unsafe {
+        from_glib_full(ffi::gst_pb_utils_get_sink_description(protocol.to_glib_none().0))
+    }
+}
+
+pub fn pb_utils_get_source_description(protocol: &str) -> Option<String> {
+    assert_initialized_main_thread!();
+    unsafe {
+        from_glib_full(ffi::gst_pb_utils_get_source_description(protocol.to_glib_none().0))
+    }
+}
+
+pub fn pb_utils_init() {
```

Comment:
What is this needed for? All functions that need this to be called first should get a guard for that